types/v56/vm_types.go

```diff
@@ -1,18 +1,21 @@
+/*
+ * Copyright 2021 VMware, Inc.  All rights reserved.  Licensed under the Apache v2 License.
+ */
 package types
 
 import "encoding/xml"
 
-// VM represents a virtual machine
+// Vm represents a virtual machine
 // Type: VmType
 // Namespace: http://www.vmware.com/vcloud/v1.5
 // Description: Represents a virtual machine.
 // Since: 0.9
-type VM struct {
+type Vm struct {
 	// Attributes
-	XMLName xml.Name `xml:"Vm"`
-	Ovf     string   `xml:"xmlns:ovf,attr,omitempty"`
-	Xsi     string   `xml:"xmlns:xsi,attr,omitempty"`
-	Xmlns   string   `xml:"xmlns,attr,omitempty"`
+	//XMLName xml.Name `xml:"Vm"`
```

Comment:
If we leave it commented out just like this, somebody will uncomment it in the future :)
What about adding a warning **why** is that done?

Reply:
added explanation